src/format/to_dom.js

```diff
@@ -109,7 +109,7 @@ class DOMSerializer {
   }
 }
 
-// :: (Node, ?Object) → DOMFragment
+// :: (Node, ?Object) → DocumentFragment
```

Comment:
I'm prefixing DOM-related types with `DOM` (also to avoid ambiguity between the DOM's `Node` and my `Node` type). I'd prefer to keep it that way.